docs/local-annotations-library-documentation.md

```diff
@@ -0,0 +1,69 @@
+# PaSh Setup and Execution Guide
+
+This guide walks you through **setting up and running PaSh**, including:
```

Comment:
We can make this a bit more specific to clarify that this walks you through setting up pash with local annotations